os/board/esp32_devkit/Kconfig

```diff
@@ -321,3 +321,20 @@ endchoice
 
 endmenu # "SPI RAM config"
 
+config ESP_LCD_ENABLE
+	bool "Enable LCD driver on ESP-WROVER-KIT board"
+	depends on ARCH_BOARD_ESP_WROVER_KIT
```

Comment:
This is a devkit board, not wroverkit. This config is unnecessary.

Reply:
Currently, wrover and devkit both are sharing os/board/esp32_devkit.
LCD feature depends on the wrover kit device.